content/survey_2022.ipynb

```diff
@@ -0,0 +1,4709 @@
+{
```

Comment:
<p>Can you make the colors for these libraries match the dominant color in the logo for each library, e.g. blue for hvPlot and red-orange for HoloViews (and green for Panel despite the recent logo color change, to keep it distinct from hvPlot)? Some like colorcet have no dominant color, but generally they do, and it would be much easier to match things up that way...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/holoviz-dev/blog/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
OK, I'll use the brand colors.. but just FYI I chose this colormap to make the Sankey a bit easier to parse and more aesthetically pleasing.. here's what it looks like now using brand colors:

![image](https://github.com/holoviz-dev/blog/assets/6613202/f3c54a9e-b5ec-4285-9d01-2d14073515bd)


vs what I had before:

![image](https://github.com/holoviz-dev/blog/assets/6613202/ee6112fa-0109-4d93-a332-c4fce0c6d5eb)

Reply:
OK, here's a compromise..  I tried to keep the hues of the colors close to the brand colors while adjusting the shades slightly to improve their harmony but maintain distinctiveness. I think this also helps avoid the dreaded Christmas combo of red on green.

![image](https://github.com/holoviz-dev/blog/assets/6613202/14710f32-2656-46a5-a617-1af408ded362)

Reply:
Perfect!
__